backend/src/test/java/com/htecgroup/skynest/utils/company/CompanyEntityUtil.java

```diff
@@ -4,8 +4,22 @@
 
 public class CompanyEntityUtil extends CompanyBasicUtil {
 
-  public static CompanyEntity getCompanyEntity() {
+  public static CompanyEntity get() {
     return new CompanyEntity(
         id, createdOn, modifiedOn, deletedOn, pib, name, address, phoneNumber, email, tierEntity);
```

Comment:
Try to make some concrete id in this class like in TierEntityUtil or in class it is extending so when we need to check in other tests if it is the correct company we can reuse this method. If you know what I mean. I had some problems in some test, so I needed to refactor some util classes so I can use existing methods for easier testing.

Reply:
Sure, I will make id from CompanyBasicUtil have a concrete value